apps/music/js/ui/title_bar.js

```diff
@@ -43,6 +59,17 @@ var TitleBar = {
     ModeManager.pop();
   },
 
+  showScanProgress: function(info) {
+    this.scanProgress.classList.remove('hidden');
+    this.scanCount.textContent = info.count;
+    this.scanArtist.textContent = info.artist || '';
+    this.scanTitle.textContent = info.title || '';
```

Comment:
@jimporter Though we display empty strings before if the info contains no data, do you think we should display strings like "Unknown album" or "Unknown artist" while scanning the new songs with no metadata tags? or maybe we can ask ux about this.